python_modules/dagster/dagster/core/definitions/repository_definition.py

```diff
@@ -634,6 +634,15 @@ def from_list(cls, repository_definitions):
                     )
                 foreign_assets[definition.key] = definition
 
+            elif isinstance(definition, AssetCollection):
+                pipelines_or_jobs["__ASSET_COLLECTION"] = build_assets_job(
```

Comment:
Missed this: should probably make a constant for this value instead of just having it in a string

Reply:
can this be a method on `AssetCollection` ? Insulates this code from details that may change 

```
asset_job = definition.build_assets_job()
pipelines_or_jobs[asset_job.name] = asset_job
```